app/models/grda_warehouse/service_history_enrollment.rb

```diff
@@ -109,43 +109,47 @@ def self.service_types
   end
 
   scope :currently_homeless, -> (date: Date.today, chronic_types_only: false) do
+    ph_scopes = [
+      entry.ongoing(on_date: date).
+        ph_move_in_after(date).
```

Comment:
`ph_move_in_after` should be `with_move_in_date_before`. Move the project type onto this chain of scopes `in_project_type(GrdaWarehouse::Hud::Project::RESIDENTIAL_PROJECT_TYPES[:ph])`